samcli/lib/providers/sam_function_provider.py

```diff
@@ -160,20 +160,29 @@ def _convert_sam_function_resource(name, resource_properties, layers, ignore_cod
         samcli.commands.local.lib.provider.Function
             Function configuration
         """
-
         codeuri = SamFunctionProvider.DEFAULT_CODEURI
+        inlinecode = None
         imageuri = None
         packagetype = resource_properties.get("PackageType", ZIP)
         if packagetype == ZIP:
-            codeuri = SamFunctionProvider._extract_sam_function_codeuri(
-                name, resource_properties, "CodeUri", ignore_code_extraction_warnings=ignore_code_extraction_warnings
-            )
-            LOG.debug("Found Serverless function with name='%s' and CodeUri='%s'", name, codeuri)
```

Comment:
We loose this debug statement for Functions with CodeUri.

Reply:
Added this back